test/parallel/test-child-process-promisified.js

```diff
@@ -12,7 +12,9 @@ const execFile = promisify(child_process.execFile);
 
   assert(promise.child instanceof child_process.ChildProcess);
   promise.then(common.mustCall((obj) => {
-    assert.deepStrictEqual(obj, { stdout: '42\n', stderr: '' });
```

Comment:
Note this is technically a breaking change — the test currently does not permit the addition of any properties. I changed this to multiple assertions, since (afaict) deepStrictEqual would be difficult to use on an instance of ChildProcess.